Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM debian:jessie
+MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
+
+RUN apt-get -qq update && apt-get -qqy install python-pip python-dev
+RUN pip install junebug
+RUN mkdir logs
+
+EXPOSE 8080
+ENTRYPOINT ["jb", "--config=/app/example-config.yaml"]
```

Comment:
@hodgestar What do you think about the idea of for now, creating a `channel-logs` folder in `example-app`, with a file so that git keeps that folder, and having the channel logs go there. And then creating a card to have junebug create the folder if it doesn't exist?